com.microsoft.java.debug.plugin/META-INF/MANIFEST.MF

```diff
@@ -16,7 +16,8 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.jdt.ls.core,
  org.eclipse.jdt.launching,
  com.google.gson;bundle-version="2.7.0",
- org.apache.commons.lang3;bundle-version="3.1.0"
+ org.apache.commons.lang3;bundle-version="3.1.0",
+ org.eclipse.lsp4j;bundle-version="0.2.0"
```

Comment:
Remove the bundle version limitation.

Reply:
`lsp4j` is a new dependency, can it be installed by `mvn install`?

Reply:
It is part of language server dependency, which is also available in plugin dependency.

Reply:
The ci server will use `mvn package` to build the jar, so it can resolve the dependency correctly. Right?

Reply:
Fixed for version issue.